cli/ballerina-cli/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -4,7 +4,7 @@ NAME
 SYNOPSIS
        bal test
        bal test [--debug] <port>
-       bal test [--offline] [--rerun-failed] [--experimental] [--observability-included]
+       bal test [--offline] [--rerun-failed] [--dump-build-time] [--experimental] [--observability-included]
```

Comment:
This is a hidden flag. So we dont need to add this here. Please check and update the build command help as well if it has this

Reply:
Removed this from all other command help files